columnflow/tasks/cms/external.py

```diff
@@ -42,13 +42,25 @@ def requires(self):
     def output(self):
         return self.target(f"weights_from_{self.data_mode}.json")
 
```

Comment:
The lines below show how to use the stagein and stageout feature. One just needs to define these methods which can return masks that, when True, decide that a certain input or output should be staged. This means that the input is moved to a local, temporary location first in the outer environment, and that outputs are first stored locally and then move to the actual remote location after the sandbox terminated (hence, in the outer environment where gfal2 usually exists).